assets/datafiles/variant_map.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
This file exists to upgrade existing users' pokemon from the old-style name to the new name. Otherwise, the app no longer has any concept of "Arceus (Normal)" as a species name.